sopel/cli/plugins.py

```diff
@@ -0,0 +1,397 @@
+        """)
+    utils.add_common_arguments(show_parser)
+    show_parser.add_argument('name', help='Plugin name')
+
+    # sopel-plugins configure <name>
+    config_parser = subparsers.add_parser(
+        'configure',
+        help="Configure plugin with a config wizard",
+        description="""
+            Run a config wizard to configure a plugin. This can be used whether
+            the plugin is enabled or not.
+        """)
+    utils.add_common_arguments(config_parser)
+    config_parser.add_argument('name', help='Plugin name')
+
+    # sopel-plugins list
+    list_parser = subparsers.add_parser(
+        'list',
+        help="List available Sopel plugins",
+        description="""
+            List available Sopel plugins from all possible sources: built-in,
+            from ``sopel_modules.*``, from ``sopel.plugins`` entry points,
+            or Sopel's plugin directories. Enabled plugins are displayed in
+            green; disabled, in red.
+        """)
+    utils.add_common_arguments(list_parser)
+    list_parser.add_argument(
+        '-C', '--no-color',
+        help='Disable colors',
+        dest='no_color',
+        action='store_true',
+        default=False)
+    list_enable = list_parser.add_mutually_exclusive_group(required=False)
+    list_enable.add_argument(
+        '-e', '--enabled-only',
+        help='Display only enabled plugins',
+        dest='enabled_only',
+        action='store_true',
+        default=False)
+    list_enable.add_argument(
+        '-d', '--disabled-only',
+        help='Display only disabled plugins',
+        dest='disabled_only',
+        action='store_true',
+        default=False)
+    list_parser.add_argument(
+        '-n', '--name-only',
+        help='Display only plugin names',
+        dest='name_only',
+        action='store_true',
+        default=False)
+
+    # sopel-plugin disable
+    disable_parser = subparsers.add_parser(
+        'disable',
+        help="Disable a Sopel plugins",
+        description="""
+            Disable a Sopel plugin by its name, no matter where it comes from.
+            It is not possible to disable the ``coretasks`` plugin.
+        """)
+    utils.add_common_arguments(disable_parser)
+    disable_parser.add_argument('name', help='Name of the plugin to disable')
+    disable_parser.add_argument(
+        '-f', '--force', action='store_true', default=False,
+        help="""
+            Force exclusion of the plugin. When ``core.enable`` is defined, a
+            plugin may be disabled without being excluded. In this case, use
+            this option to force its exclusion.
+        """)
+    disable_parser.add_argument(
+        '-r', '--remove', action='store_true', default=False,
+        help="""
+            Remove from ``core.enable`` list if applicable.
+        """)
+
+    # sopel-plugin enable
+    enable_parser = subparsers.add_parser(
+        'enable',
+        help="Enable a Sopel plugins",
```

Comment:
You made it _this far_ without running into your nemesi**s**!

```suggestion
        help="Enable a Sopel plugin",
```